### PR TITLE
Chore: Added new translations

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the details for making this change. What existing problem does the pull request solve?**
Adds some translations from the translation team. And adds a missing snap shot

**Related github/jira issue (required)**:
NA

**Steps necessary to review your pull request (required)**:
- if all checks pass, this can be merged

### CODE REVIEW

File: src/components/ids-locale/cultures/bg-messages.js [hunk 1]
```diff
@@ -148,7 +149,7 @@ const messages = {
   ExamplePlaceholder: { id: 'ExamplePlaceholder', value: 'Примерен контейнер', comment: 'Translated text for an example form placeholder text' },
   Export: { id: 'Export', value: 'Експортиране', comment: 'Export tooltip' },
   ExportToExcel: { id: 'ExportToExcel', value: 'Експортирай в Excel', comment: 'Export To Excel menu option in datagrid' },
-  ExtraSmall: { id: 'ExtraSmall', value: 'Много малки', comment: 'A smaller font size and compressed mode for datagrid rows.' },
+  ExtraSmall: { id: 'ExtraSmall', value: 'Много малки', comment: 'A smaller font size and compressed mode for datagrid rows. NOTE: Please change french to Très petite (Masculine form)' },
```
Comment (vivz753) [inline src/components/ids-locale/cultures/bg-messages.js:152]: Has a ticket been made for the `NOTE: Please change french to Très petite (Masculine form)`?

**Review by vivz753 — DISMISSED** [review]

File: test/ids-list-builder/ids-list-builder-func-test.js.snap [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`IdsListBuilder Component renders correctly 1`] = `
+"<ids-list-builder sortable=\\"true\\" item-height=\\"46\\">
+  </ids-list-builder>"
+`;
```
Comment (vivz753) [inline test/ids-list-builder/ids-list-builder-func-test.js.snap:6]: Was committing this necessary? I prefer to keep snaps out of the repo

File: src/components/ids-locale/cultures/bg-messages.js (hunk 1 above)
Comment (tmcconechy) [reply src/components/ids-locale/cultures/bg-messages.js:152]: yes its https://github.com/infor-design/enterprise/issues/4336 - these are shared between both projects now so getting fixes from either project. However. I will remove the note for the translators now and push

File: test/ids-list-builder/ids-list-builder-func-test.js.snap (hunk 2 above)
Comment (tmcconechy) [reply test/ids-list-builder/ids-list-builder-func-test.js.snap:6]: Yes all these snaps should be in the repo or else you dont know if it changed. i guess we didnt notice this until i spotted it. Then you see what happens is that every build it makes a new one and does no comparison.

From https://jestjs.io/docs/snapshot-testing "The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process"

**Pull request merged**